Server/Temp.Domain/EventMessaging/Contracts/IDomainEventBus.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ModularHouse.Server.Temp.Domain.EventMessaging.Contracts;
+
+public interface IDomainEventBus
+{
+    Task PublishAsync<TEvent>(TEvent domainEvent) where TEvent : IDomainEvent;
+    Task<Guid> SubscribeAsync<TEvent>(Action<TEvent> callback) where TEvent : IDomainEvent;
+    Task UnsubscribeAsync(Guid subscriptionId);
+    
+    Task<DomainEventWaitResult<TEvent>> WaitAsync<TEvent>(int milliseconds, Guid? transactionId = null)
+        where TEvent : IDomainEvent;
```

Comment:
For now I see this flow like that:
we publish event => call waitAsync method (here is handle, execution, unsubscribe) 

What do you think if we create HandleAsync method and set flow:
on the startup of our application we subscribe to all necessarry topics and open data pull cycle (every 300 ms pull the data for new messages)
so when we publish event => it automaticaly call HandleAsync method for specific handler, that subscribed to the specific topic.

Benefit: we shouldn't call this method manually, it'll call automaticaly.

Reply:
Discussed in a call.